bundles/org.openhab.binding.cul/src/main/resources/OH-INF/thing/cul_bridge.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="maxcul"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="cul_max_bridge">
+		<label>CUL Bridge</label>
+		<description>This bridge is used when using a CUL (serial) in MORIZ mode (MAX!)</description>
+		<channels>
+			<channel id="led" typeId="onoff"/>
+			<channel id="credit10ms" typeId="credits"/>
+		</channels>
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<label>Serial port</label>
```

Comment:
All labels should have (mostly) uppercase words. See specific label formatting: https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
				<label>Serial Port</label>
```